cardano-testnet/src/Testnet/Process/Cli/Transaction.hs

```diff
@@ -35,6 +39,45 @@ data TxBody
 
 data SignedTx
 
+-- | Calls @cardano-cli@ to sign a simple ADA transfer transaction using
+-- the specified key pairs.
+-- This function takes five parameters:
+-- 
+-- Returns the generated @File TxBody In@ file path to the created unsigned
+-- transaction file.
+buildTransferTx
+  :: Typeable era
+  => H.MonadAssertion m
+  => MonadTest m
+  => MonadCatch m
+  => MonadIO m
+  => H.ExecConfig -- ^ Specifies the CLI execution configuration.
+  -> EpochStateView -- ^ Current epoch state view for transaction building. It can be obtained
+                    -- using the 'getEpochStateView' function.
+  -> ShelleyBasedEra era -- ^ Witness for the current Cardano era.
+  -> FilePath -- ^ Base directory path where the unsigned transaction file will be stored.
+  -> String -- ^ Prefix for the output unsigned transaction file name. The extension will be @.txbody@.
+  -> PaymentKeyInfo -- ^ Payment key pair used for paying the transaction.
+  -> PaymentKeyInfo -- ^ Payment key of the recipient of the transaction.
+  -> Int -- ^ Amount of ADA to transfer (in Lovelace).
```

Comment:
`Coin` would be a better type to use here.

Reply:
Solved [here](https://github.com/IntersectMBO/cardano-node/commit/89269b20578934f3797fb11e461ed3227360863d)